CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 # TiSpark Changelog
 All notable changes to this project will be documented in this file.
 
+## [TiSpark 2.2.0] 2019-08-30
+### New Features
+* Natively support writing data to TiKV using Spark Data Source API
+* Support select from partition table [#916](https://github.com/pingcap/tispark/pull/916)
+* Release one tispark jar (both support Spark-2.3.x and Spark-2.4.x) instead of two [#933](https://github.com/pingcap/tispark/pull/933)
+* Add spark version to tispark udf ti_version [#943](https://github.com/pingcap/tispark/pull/943)
```

Comment:
s/Add spark version to tispark udf ti_version/Add spark version to tispark's udf ti_version